.github/workflows/refapp-2x-person.yml

```diff
@@ -0,0 +1,40 @@
+name: RefApp 2.x Person
```

Comment:
Is it possible to have this consumed in findPatientSteps.

Reply:
i doubt if they can. but am open to advice

Reply:
@sherrif10 don't you think https://issues.openmrs.org/browse/RATEST-301 will cater for what you instead want me to do here? because i believe the attached ID can be added to the findPatientSteps

Reply:
Sorry , we can go with this approach since there seem to be slightly different.